helper/ip.c

```diff
@@ -28,3 +28,23 @@ int odph_ipv4_addr_parse(uint32_t *ip_addr, const char *str)
 
 	return 0;
 }
+
+int odph_ipv6_addr_parse(uint8_t * const ip_addr, const char * const str)
+{
+	int i, j;
+	uint16_t hextet[8];
+
+	if ((NULL == ip_addr) || (NULL == str))
+		return -1;
+
+	if (sscanf(str, "%hx:%hx:%hx:%hx:%hx:%hx:%hx:%hx", &hextet[0], &hextet[1], &hextet[2],
```

Comment:
"hx" is not portable for scanning into uint16_t as there is no guarantee that unsigned short int is the same as uint16_t. You should use the format specifier defined by the SCNx16 macro.

Reply:
Yes.. I will fix this in another change-set. I can completely avoid using sscanf() and just parse it like a string. Is that fine?